packages/editor/src/components/char-count/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { _x } from '@wordpress/i18n';
+import { count as charCount } from '@wordpress/wordcount';
+
+function CharCount( { content } ) {
+	/*
+	 * Only count the number of characters (no spaces).
+	 * Do not translate into your own language.
+	 */
+	const charCountType = _x( 'characters_excluding_spaces', 'Word count type. Do not translate!' );
```

Comment:
I see #13225. Since the default language is English, I think it's better to change this to  `characters_including_spaces`.

```suggestion
	const charCountType = _x( 'characters_including_spaces', 'Character count type. Do not translate literally!' );
```

Reply:
Done